recipes/pangolin/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "pangolin" %}
```

```suggestion
{% set name = "pangolin-opengl" %}
```

As a `pangolin` recipe already exists for a different program at bioconda and bioconda folks use `conda-forge` preferentially over `bioconda` the name has to be changed here as this is a different tool.

Would you consider changing the name to for example `pangolin-opengl` or something similar?